app/javascript/packs/utils/calendar.js

```diff
@@ -30,9 +30,9 @@ export function weeks(start, range){
   });
 }
 
-export function innerRange(base){
-  return [base.clone().subtract(1, 'M').date(base.date()),
-          moment.min(base.clone().add(1, 'd'), moment().add(1, 'd'))];
+export function innerRange(base, currentDate=current()){
+  return [base.clone(),
+          moment.min(base.clone().add(1, 'M').subtract(1, 'd'), currentDate)];
```

Comment:
I have never seen a function being called in the argument of another function to determine the default value of an argument. I think you can make it more explicit and readable:
```suggestion
export function innerRange(base, currentDate){
  const validCurrentDate = currentDate || current();

  return [base.clone(),
          moment.min(base.clone().add(1, 'M').subtract(1, 'd'), validCurrentDate)];
```

Reply:
Btw, looks like we don't have a test to guarantee we have a default date for `currentDate`. If we remove the default value, all the tests will still pass. Shouldn't we have a test for that?
![Peek 2022-08-26 15-17](https://user-images.githubusercontent.com/54550926/186967308-381366ed-88ff-4111-b386-944e9e12c9b5.gif)